cmd/serverless-init/cloudservice/appservice_test.go

```diff
@@ -19,11 +19,23 @@ func TestGetAppServiceTags(t *testing.T) {
 	t.Setenv("APPSVC_RUN_ZIP", "false")
 
 	tags := service.GetTags()
+	tags["aas.environment.os"] = "test_os"
```

Comment:
Feels hacky but doing the work of detecting the OS and runtime of what the tests are running on feels pointless, is this alright?